ui/src/App.tsx

```diff
@@ -3,23 +3,78 @@ import './App.sass';
 import logo from './components/assets/Mizu-logo.svg';
 import {Button} from "@material-ui/core";
 import {HarPage} from "./components/HarPage";
+import Tooltip from "./components/Tooltip";
+import {makeStyles} from "@material-ui/core/styles";
 
 
+const useStyles = makeStyles(() => ({
+    tooltip: {
+        backgroundColor: "#3868dc",
+        color: "white",
+        fontSize: 13,
```

Comment:
I think we usually use css or sass for styles.

Reply:
I think we are doing it like this sometimes when it is done using useStyles